packages/vscode-dependencies-validation/test/vscodeMocks.ts

```diff
@@ -0,0 +1,25 @@
+import { NPM_DEPENDENCY_ISSUES_CODE } from "../src/constants";
```

Comment:
is this file used?
it does not seem to be rendered as TypeScript by github for some reason